utils/build-script

```diff
@@ -1186,7 +1186,7 @@ Using option presets:
   using the name=value syntax on the command line.
 
 
-Any arguments not listed are forwarded directly to Swift's
+Any arguments not listed, or arguments after `--` are forwarded to Swift's
```

Comment:
Yep. It used to be true, but we changed it once we decided to move build-script-impl into build-script. (That effort itself then ran into issues, but that's another story.)